code/Player/Roles/Role.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace ssl.Player.Roles
+{
+    /// <summary>
+    /// Player's role
+    /// </summary>
+    public abstract class Role
+    {
+        public abstract string Id { get; }
+        public abstract string Name { get; }
+        public abstract string Description { get; }
+        public virtual string Model => "models/units/simpleterry.vmdl";
```

Comment:
Avec seulement Model on sait pas trop de quel model tu parle (personnage, vêtements ou autre) je trouve
```suggestion
        public virtual string CharacterModel => "models/units/simpleterry.vmdl";
```